test/tests.js

```diff
@@ -241,8 +241,29 @@ QUnit.test('false secure value', function (assert) {
 	assert.strictEqual(actual, expected, 'false should not modify path in cookie string');
 });
 
+QUnit.test('unofficial attribute', function (assert) {
+	assert.expect(1);
+	assert.strictEqual(Cookies.set('c', 'v', {
+		unofficial: 'anything'
+	}), 'c=v; path=/; unofficial=anything', 'should write the cookie string with unofficial attribute');
+});
+
+QUnit.test('unofficial attribute with boolean value', function (assert) {
+	assert.expect(1);
+	assert.strictEqual(Cookies.set('c', 'v', {
+		unofficial: true
```

Comment:
Without boolean support:
```
   raw     gz Compared to master @ 7fd36e66b60a4e0ac9d09207fb3a348a2969152c
  +106    +61 build/js.cookie-2.1.3.min.js
  +321    +90 src/js.cookie.js
```

So it's only 9-10 bytes less. Do you thing it is worth?